state-chain/runtime/src/chainflip.rs

```diff
@@ -969,3 +970,23 @@ impl FetchesTransfersLimitProvider for EvmLimit {
 		Some(20)
 	}
 }
+
+pub struct SolanaContractSwapper;
+impl SolanaContractSwap for SolanaContractSwapper {
+	fn initiate_contract_swap(
+		from: Asset,
+		to: Asset,
+		deposit_amount: AssetAmount,
+		destination_address: EncodedAddress,
+		tx_hash: TransactionHash,
+	) {
+		let _ = SolanaIngressEgress::contract_swap_request(
+			RuntimeOrigin::root(),
```

Comment:
I think you can do something like this: 

RuntimeOrigin::from(pallet_cf_witnesser::Origin::CurrentEpochWitnessThreshold
```suggestion
			pallet_cf_witnesser::Origin::CurrentEpochWitnessThreshold.into(),
```